pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -261,6 +301,7 @@ func (o *OvirtMapper) MapDataVolumes(targetVMName *string) (map[string]cdiv1.Dat
 		}
 		quantity, _ := resource.ParseQuantity(diskSizeConverted)
 		accessMode := o.getAccessMode(diskAttachment)
+		volumeMode := o.getVolumeMode(disk, o.mappings)
```

Comment:
To detect the volume mode we're iterating over the same mappings slice for the second time (first - for the storage class in `getStorageClassForDisk`). In both cases we should get exactly the same mapping instance as a base for retrieving both storage class and volume mode. What do you think about finding the mapping first and than passing it to `getVolumeMode` and `getStorageClassForDisk` to extract the needed parameter?

Reply:
Good point, thanks.